src/components/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { forwardRef, useRef } from "react";
+import cx from "classnames";
+import { useMergeRefs } from "../../hooks";
+import VibeComponentProps from "../../types/VibeComponentProps";
+import VibeComponent from "../../types/VibeComponent";
+import { getTestId } from "../../tests/test-ids-utils";
+import { ComponentDefaultTestId } from "../../tests/constants";
+import styles from "./EmptyState.module.scss";
+import Flex from "../Flex/Flex";
+import Heading from "../Heading/Heading";
+import Text from "../Text/Text";
+import Button from "../Button/Button";
+
+export interface EmptyStateProps extends VibeComponentProps {
+  imgSrc: string;
+  title: string;
+  body: string;
```

Comment:
Not sure if `body` is the best name for it... Maybe just `text`, `message` or smth else?

Reply:
content?